style.css

```
@@ -0,0 +1,333 @@
+}
+
+.header__button {
+    border-radius: 8px;
+    width: 128px;
+    height: 48px;
+    padding: 14px 23px;
+    color: #F3F4F6;
+}
+
+
+/*banner section*/
+.banner-section {
+    background-color: #CFE5FF;
+}
+
+.banner {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    height: 540px;
+    gap: 7px;
+}
+
+.banner__title {
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    gap: 32px;
+}
+
+.banner__text {
+    font-weight: 700;
+    font-size: 40px;
+    color: #374151; 
+    line-height: 56px;
+}
+
+
+.banner__button {
+    border-radius: 40px;
+    padding: 16px 124px;
+    width: 357px;
+    height: 56px;
+    color: #F9FAFB;
+    font-size: 20px;
+    margin-bottom: 110px;
+}
+
+.bottom-banner {
+    margin-top: 138px;
+}
+
+
+.banner__img {
+    width: 60vw;
+    max-width: 746px;
+    height: auto;
+}
+
+.button {
+    display: inline-block;
+    background-color: #3692FF;
+    font-weight: 600;
+    text-align: center;
+}
+
+.button:hover {
+    background-color: #1967D6;
+}
+
+
+/*card section*/
+.card-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-block: 138px;
+}
+
+.card {
+    display: flex;
+    align-items: center;
+    width: 988px;
+    height: 437px;
+    background-color: #FCFCFC;
+    border-radius: 12px;
+    gap : 64px;
+}
+
+.card__img {
+    max-width: 579px;
+    width: 100%;
+    height: 100%;
+}
+
+.card__text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 12px;
+    width: auto;
+    height: 238px;
+    white-space: nowrap;
+}
+
+.card__category {
+    font-size: 18px;
+    color: #3692FF;
+    font-weight: 700;
+}
+
+.card__title {
+    font-size: 40px;
+    font-weight: 700;
+    color: #374151;
+    line-height: 56px;
+}
+
+.card__description {
+    font-size: 24px;
+    font-weight: 500;
+    color: #374151;
+    line-height: 32px;
+}
+
+.card--reverse {
+    flex-direction: row-reverse;
+    text-align: right;
+}
+
+.banner__title--center {
+    align-self: center;
+    padding-top: 30px;
+}
+
+/*footer section*/
+.footer{
+    display: flex;
+    justify-content: space-around;
+    background-color: #111827;
+    padding-top: 32px;
+    height: 160px;
+    font-size: 16px;
+    font-weight: 400;
+}
+
+.footer__name{
+    color: #9CA3AF; 
+}
+
+.footer-links{
+    display: flex;
+    gap: 30px;
+}
+
+.footer__link{
+    color: #E5E7EB;
+}
+
+.footer-icons{
+    display: flex;
+    gap: 12px; 
+}
+
+/*Laptop Version*/
+@media (max-width: 1024px) {
+
+   .herader-nav {
+        margin-inline: 10%;
+    }
+    
+    .banner {
+        height: 450px;
+    }
+
+    .banner__button {
+        width: 100%;
+        white-space: nowrap;
+        text-align: center;
+        padding: 16px 96px;
+    }
+
+    .card {
+        max-width: 760px;
+        height: 437px;
+        gap: 48px;
+    }
+
+    .card__img {
+        width: 60%;
+    }
```

❗️ 수정요청
화면크기가 줄어들때, width가 줄어들고, height는 100%로 유지되니 이미지가 왜곡되어 보여요~
이미지는 최대한 기존 비율을 유지하는 것이 좋으니 height를 다른 값으로 변경하시는 것을 추천드려요.

```suggestion
    .card__img {
        width: 60%;
        height: auto;
    }
```